api.planx.uk/modules/webhooks/routes.ts

```diff
@@ -53,6 +55,12 @@ router.post(
   sanitiseApplicationDataController,
 );
 
+router.post(
+  "/webhooks/hasura/validate-input/jsonb/clean-html",
```

Comment:
Open to thoughts on the URL path here - I figured having a column type parameter would handle us possibly expanding this in future, e.g - 

 - `/webhooks/hasura/validate-input/text/clean-html`
 - `/webhooks/hasura/validate-input/number/something`